src/app/modules/submission/encrypt-submission/encrypt-submission.utils.ts

```diff
@@ -359,3 +371,92 @@ export const getPaymentIntentDescription = (
     }
   }
 }
+
+/**
+ * Determines the prefix for a question based on whether it is verified
+ * by MyInfo.
+ * @param response
+ * @param hashedFields Field ids of hashed fields.
+ * @returns the prefix
+ */
+const getMyInfoPrefix = (
+  response: ProcessedFieldResponse,
+  hashedFields: Set<MyInfoKey>,
+): string => {
+  return !!response.myInfo?.attr && hashedFields.has(response._id)
+    ? MYINFO_PREFIX
+    : ''
+}
+
+export const getAnswersForChild = (
```

Comment:
`getAnswersForChild` and `getMyInfoPrefix` can be shared with `email`

Reply:
extracted them into submission.utils [cf430b0](https://github.com/opengovsg/FormSG/pull/6870/commits/cf430b06b9cf302b5665c1690b91a7fd89c67d87)